build-tests/x86/suse/test-image-disk-ramdisk/appliance.kiwi

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="7.3" name="kiwi-test-image-disk-ramdisk">
+    <description type="system">
+        <author>Marcus Schäfer</author>
+        <contact>ms@suse.com</contact>
+        <specification>Disk test build, deploy to ramdisk</specification>
+    </description>
+    <preferences>
+        <version>1.42.1</version>
+        <packagemanager>zypper</packagemanager>
+        <locale>en_US</locale>
+        <keytable>us</keytable>
+        <timezone>Europe/Berlin</timezone>
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <rpm-check-signatures>false</rpm-check-signatures>
+        <bootsplash-theme>bgrt</bootsplash-theme>
+        <bootloader-theme>openSUSE</bootloader-theme>
+        <type image="oem" filesystem="ext4" firmware="uefi" installiso="true" installboot="install" kernelcmdline="rd.kiwi.ramdisk ramdisk_size=8192000 console=ttyS0">
+            <size unit="G">4</size>
```

Comment:
Why? I'd prefer having some smaller tests if possible, what about 2G, it will definitely be enough for a JeOS.

Reply:
remember you need some space to work with the image. It's all in ram this means if the image has ~2G of files you don't have anything left for the RAM of the OS. That's why I used 4G of image file size which in fact means 2G are free to work with the system. But for the purpose of the test it's ok to go down. I try with 2G but I think I know why I selected 4G in the end